### DISCUSSION TITLE
Sidebar with multiple level

### DISCUSSION THREAD

**Discussion by hoanga4bg** [themesberg/flowbite-react#1043] [Q&A]
Hello everyone, I want to create a sidebar with many levels like below.
![image](https://github.com/themesberg/flowbite-react/assets/55157047/4ebbf45b-9161-4f73-8f1e-ddad04b2f60a)

I tried adding <Sidebar.Collapse> into another <Sidebar.Collapse> but it didn't work.
Does anyone know how to do it?

**Comment by rluders** [comment]
I think that it isn't supported. Not even by the standard library.

**Reply by MiroslavPetrik** [reply]
@rluders Is it a no-go to add anything on top of the standard flowbite? 
E.g. I also see some improvements I could add here and there.

For example, the `<Progress />` sizing starts with `sm` but I have a usecase where I would have `xs` and would happily ship a PR for that.
I saw multiple times this comment, that something is out-of-scope as it is not part of the standard. I understand that there is lots of work to get to 1.0 (which I assume will cover the standard flowbite).

So can I consider the non-standard features/request to have the lowest priority, but possibly mergeable when I provide a PR.
OR should I forget about it, and only do such improvements/extensions in "user land"?
